src/components/User/Profile.vue

```diff
@@ -39,13 +39,21 @@
         class="map"
       />
       <div
-        class="info-item"
+        class="info-item inlinee"
         style="white-space: nowrap; overflow: hidden; text-overflow: ellipsis">
         <strong>
           <i class="fa fa-fw fa-envelope-o on-left" />
         </strong>
         <a :href='"mailto:" + user.email'>{{ user.email }}</a>
       </div>
+      <q-btn
+        icon="fa-pencil"
+        small
+        round
+        class="inlinee on-right"
+        color="secondary"
+        @click="$router.push('/settings')"
```

Comment:
I missed this. We prefer named routes instead of referencing the URL directly

Reply:
Okay. I will try to resolve this.